specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -37,6 +37,16 @@
           "items": {
             "type": "object"
           }
+        },
+        "folder": {
+          "description": "The folder that this Dataset is in.",
```

Comment:
If this property is omitted, is there a default folder that is used? If so, it may be worth mentioning.

Reply:
All the entities will appear at the root level unless a folder is specified. If that is worth mentioning then i can add that in

Reply:
Your call :)

Reply:
Changed!